### PR TITLE
STCOM-863 Upgrade to Storybook v6, upgrade postcss stack...

### PR DESCRIPTION
**Pull request by JohnC-80**
Further upgrades punted until webpack v5/postcss v8/postcss-loader v5  upgrade - but spanning some multiple major versions in this go around...
Punting deprecated 'knobs' functionality in the component stories.
Reformatting 'preview' to suit what new storybook wants.

Depends on the storybook-6 branch of `stripes-webpack`: https://github.com/folio-org/stripes-webpack/pull/24

### CODE REVIEW

**Review by rasmuswoelk — APPROVED** [review]
LGTM :)

**Review by zburke — APPROVED** [review]
LGTM now! In concert with the [`storybook-6` branch on `stripes-webpack`](https://github.com/folio-org/stripes-webpack/pull/24), all app icons appear correctly.

File: .storybook/stcom-webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+    // Stage 3
+    '@babel/plugin-syntax-import-meta',
+    ['@babel/plugin-proposal-class-properties', { loose: true }],
+    ['@babel/plugin-proposal-private-methods', { loose: true }],
+
+    // Others
+    'react-hot-loader/babel',
+  ]
+};
```
Comment (zburke) [inline .storybook/stcom-webpack.config.js:34]: Instead, could `babel-options` be imported via the [`@folio/stripes-cli` re-export of babel-options](https://github.com/folio-org/stripes-cli/blob/1d34cef8e63e7243f011ebdaf083421a0181bf0c/index.js) that is contained in [`stripes-webpack`](https://github.com/folio-org/stripes-webpack/blob/f04d4770705894b029639a11e52f4ec762944b0e/webpack/babel-options.js)?

Comment (JohnC-80) [reply .storybook/stcom-webpack.config.js]: 👍 Most definitely - Done!! thanks!!!

**Pull request merged**